kong/init.lua

```diff
@@ -653,12 +634,28 @@ function Kong.init_worker()
 
   kong.db:set_events_handler(worker_events)
 
-  ok, err = load_declarative_config(kong.configuration,
-                                    declarative_entities,
-                                    declarative_meta)
-  if not ok then
-    stash_init_worker_error("failed to load declarative config file: " .. err)
-    return
+  if kong.configuration.database == "off" then
+    -- databases in LMDB need to be explicitly created, otherwise `get`
+    -- operations will return error instead of `nil`. This ensures the default
+    -- namespace always exists in the
+    local t = lmdb_txn.begin(1)
+    t:db_open(true)
+    assert(t:commit())
```

Comment:
Should this follow the same error-handling pattern as the surrounding code (check return value and call `stash_init_worker_error(...)` if needed) instead of `assert()`?

Reply:
This has been fixed already.